fabric-gamerule-api-v1/src/main/java/net/fabricmc/fabric/api/gamerule/v1/CustomGameRuleCategory.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.gamerule.v1;
+
+import net.minecraft.text.Text;
+import net.minecraft.util.Identifier;
+
+/**
+ * Utility class for registering GameRule objects with custom categories outside of the categories Minecraft provides.
+ */
+public final class CustomGameRuleCategory {
+	public static CustomGameRuleCategory of(Identifier id, Text name) {
+		return new CustomGameRuleCategory(id, name);
+	}
+
+	private final Identifier id;
```

Comment:
Why is there no getter here?

Reply:
I don't really expose a way to get the custom category from the rule currently. Which would probably make sense in that case.